apps/ledger-live-desktop/src/renderer/modals/AddAccounts/steps/StepChooseCurrency.tsx

```diff
@@ -227,6 +230,9 @@ export const StepChooseCurrencyFooter = ({
     } else if (parentCurrency) {
       // set parentCurrency in already opened add account flow and continue
       setCurrency(parentCurrency);
+      if (mock) {
+        window.mock.events.mockDeviceEvent({ type: "opened" });
```

Comment:
The modification here fixs the following issue:
when we launch the app with:
MOCK=1 pnpm dev:lld

Add a currency in LLD, the app is stuck in the device step

Reply:
I think you should be doing this `mockDeviceEvent` in the actual tests

Reply:
It is not related to tests. The objective here is to provide mock accounts when developer runs LLD with MOCK mode without connecting device.

Reply:
code removed